ironfish/src/network/peerNetwork.ts

```diff
@@ -1361,15 +1358,11 @@ export class PeerNetwork {
       return
     }
 
-    if (this.node.memPool.acceptTransaction(transaction)) {
-      this.onTransactionAccepted.emit(transaction, received)
-    }
-
-    // Check 'exists' rather than 'accepted' to allow for rebroadcasting to nodes that
-    // may not have seen the transaction yet
-    if (this.node.memPool.exists(transaction.hash())) {
```

Comment:
Not checking the result of this function anymore has the interesting result of gossiping transactions with overlapping nullifiers in the mempool (and lower fee rate). This could be a good think since not gossiping these transactions seemed opinionated about which transactions other nodes would prefer

Reply:
Deciding to continue to not gossip transactions with overlapping nullifiers and lower fees. Looked into btc-core and go-eth and both seem to not send out lower fee conflicting transactions